crates/typst-cli/src/args.rs

```diff
@@ -386,6 +386,12 @@ pub struct FontsCommand {
 #[derive(Debug, Clone, Parser)]
 pub struct FontArgs {
     /// Adds additional directories to search for fonts
```

Comment:
```suggestion
    /// Adds additional directories that are recursively searched for fonts
```
maybe this will help with the person having this problem:
> Which directory level should this PATH point to, 'xxx/share/fonts/opentype/SourceCodePro-Black.otf' or 'xxx/share/fonts/opentype' or 'xxx/share/fonts' ?